canton/BUILD.bazel

```diff
@@ -23,7 +25,7 @@ cat << EOF > $@
 package com.digitalasset.canton.buildinfo
 
 case object BuildInfo {{
-  val version: String = "{sdk_version}"
+  val version: String = "2.7.0-SNAPSHOT"
   val scalaVersion: String = "{scala_version}"
   val sbtVersion: String = "bazel"
   val damlLibrariesVersion: String = "2.7.0-SNAPSHOT"
```

Comment:
Here as well ? (2.8)

Reply:
Actually used {sdk_version} as that's the version we're compiling with.